crypto/fipsmodule/kem/kem.c

```diff
@@ -169,27 +169,23 @@ DEFINE_LOCAL_DATA(KEM, KEM_ml_kem_1024) {
 }
 
 const KEM *KEM_find_kem_by_nid(int nid) {
-
   switch (nid) {
     case NID_MLKEM512:
       return KEM_ml_kem_512();
     case NID_MLKEM768:
       return KEM_ml_kem_768();
     case NID_MLKEM1024:
       return KEM_ml_kem_1024();
+    // Try legacy KEMs.
+    case NID_KYBER512_R3:
+      return get_legacy_kem_kyber512_r3();
+    case NID_KYBER768_R3:
+      return get_legacy_kem_kyber768_r3();
+    case NID_KYBER1024_R3:
+      return get_legacy_kem_kyber1024_r3();
```

Comment:
Minor Nit: Should we name these something like `KEM_legacy_kem_kyber512_r3` to align with the rest of our `KEM_*` naming scheme?

Reply:
Will be removed at some point when can retire the now old Kyber implementation.